src/components/List/List.jsx

```diff
@@ -59,38 +108,35 @@ export const List = ({ token }) => {
             </button>
           </div>
           <ul className="collection-list">
-            {value.docs
+            {items
               .filter((doc) =>
-                doc
-                  .data()
-                  .item.toLowerCase()
-                  .includes(filterText.toLowerCase()),
+                doc.item.toLowerCase().includes(filterText.toLowerCase()),
               )
               .map((doc) => (
-                <li key={doc.id}>
+                <li key={doc.id} className={`${setColor(doc)}`}>
```

Comment:
Nit: this function returns strings, so you don't need to wrap it in a template string.

```suggestion
                <li key={doc.id} className={setColor(doc)}>
```